README.md

```diff
@@ -93,3 +93,7 @@ muscle: 1
 action: 'move to'
 setpoint: 10.0" 
 ```
+### Units
+- Position controller ('move to' command) expects the setpoint in radians 
+- actuarorPosition and tendonDisplacement are displayed in encoder ticks
+- 1 rad ~ 18000 ticks
```

Comment:
from the yaml file: rad_per_encoder_count: 0.00005788606746738269
ticks per rad = 1/ 0.00005788606746738269 ~ 18000